scripts/extract-strings.js

```diff
@@ -74,7 +74,7 @@ function runner(err, files) {
     const messages = files.map(file => new Promise(resolve => fs.readFile(file, 'utf8', (err, contents) => resolve(extractMessages(contents)))));
     Promise.all(messages).then(values => {
         const allMessages = values.reduce((all, current) => {
-            return Object.assign({}, all, current);
+            return Object.assign(Object.assign({}, all), current);
```

Comment:
Huh. What changed that required this change?

Reply:
I do not know. I updated from a much older version of TS.